images/plbase/python-requirements.txt

```diff
@@ -1,44 +1,44 @@
 ansi2html==1.9.2
 automata-lib==9.0.0
 chevron==0.14.0
-coloraide==4.6
-Faker==37.3.0
-lxml==5.4.0
-matplotlib==3.10.1
+coloraide==4.7.2
+Faker==37.4.0
+lxml==6.0.0
```

Comment:
This is the only major version bump here. I want to see if this passes in CI. We have pretty comprehensive tests for our usage of this in question rendering, so I'm hopeful that will surface any issues.

The release notes at https://github.com/lxml/lxml/releases/tag/lxml-6.0.0 don't call out anything particularly concerning.

Reply:
This did end up causing a test failure, which was actually an issue in the test itself: https://github.com/PrairieLearn/PrairieLearn/pull/12305